.github/workflows/cypress.yml

```
@@ -1,5 +1,9 @@
-name: Cypress - Pull Request
-on: [pull_request]
+name: Cypress
+on: 
+  push:
+    branches_ignore:
```

```suggestion
    branches-ignore:
```
If only I could get things right on the first try 😉